includes/admin/class-wcmp-export.php

```diff
@@ -1168,11 +1168,11 @@ public function getItemWeightInGram(int $weight): float
     }
 
     /**
-     * @param int $item
+     * @param object $item
      *
      * @return float
      */
-    public static function getItemWeight_kg(int $item): float
+    public static function getItemWeight_kg($item): float
```

Comment:
Ik weet dat je dit niet heb aangepast maar dit mag je best refactoren (ook op alle plekken waar de method wordt gebruikt)

```suggestion
    public static function getItemWeightKg($item): float
```

Reply:
done